src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.controller.js

```diff
@@ -61,4 +63,12 @@ function calendarVideoconferenceFormController(uuid4, VideoConfConfigurationServ
       self._videoconference !== EVENT_VIDEOCONFERENCE_OPTIONS.NO_VIDEOCONFERENCE) ?
       self.videoConfHostname + self._roomName : undefined;
   }
+
+  function onLinkCopySuccess() {
+    notificationFactory.weakSuccess('', 'Video conference link has been copied successfully');
+  }
+
+  function onLinkCopyError(error) {
+    $log.error('cannot copy video conference link', error);
```

Comment:
If you've decided to use a notification to inform the user when the link has been copied successfully, you should also use a notification to inform the user in case there's an error.